tests/API_RH.py

```diff
@@ -0,0 +1,48 @@
+# Import and load necessary lib
+from bs4 import BeautifulSoup
+import requests
+
```

Comment:
can you verify that all the functions in this page work?

Reply:
I did, Also pytest works fine